manager/src/test/java/com/redhat/service/smartevents/manager/api/user/SchemaAPITest.java

```diff
@@ -27,7 +27,7 @@
 @QuarkusTest
 public class SchemaAPITest {
 
-    private static final List<String> availableActions = List.of("kafka_topic_sink_0.1", "send_to_bridge_sink_0.1", "slack_sink_0.1", "webhook_sink_0.1");
+    private static final List<String> availableActions = List.of("kafka_topic_sink_0.1", "send_to_bridge_sink_0.1", "slack_sink_0.1", "webhook_sink_0.1", "aws_lambda_sink_0.1");
```

Comment:
test fails, number of catalog items is 8 now:
https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/pull/825/files#diff-45b9d82dd36a398f840534c9348f47bb1b60e04402ea5cfb853d7333fd454dbeR57

Reply:
done